### ISSUE TITLE
[Accessibility]A11y_VSCode_WelcomeScreen_ScreenReader: JAWS reads "Clickable" after each heading present on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51972]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
Screenreader Versions: JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Welcome screen.
3. Read all the headings- "Start","Help","Customize" and "Learn","Tools and Languages" and "Settings and Keybindings".

**Actual:**
JAWS reads all the heading names along with word "clickable".

**Expected:**
JAWS should only read the headings on the screen. "Clickable" is irrelevant word repeated after each heading.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not get exact headings present on the screen and repeating irrelevant words would be irritating to the user. 

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_WelcomeScreen_JAWS_Headings.pptx](https://github.com/Microsoft/vscode/files/2105233/A11y_VSCode_WelcomeScreen_JAWS_Headings.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
We do not support jaws
